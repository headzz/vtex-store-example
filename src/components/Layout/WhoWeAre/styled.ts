import { colors } from 'configuration'
import styled from 'styled-components'

export const WhoWeAreStyled = styled.section`
  background: linear-gradient(
    90deg,
    ${colors.WHITE} 50%,
    ${colors.GREY_LVL1} 50%
  );
  padding: 6.4rem 0;
  width: 100%;
`
