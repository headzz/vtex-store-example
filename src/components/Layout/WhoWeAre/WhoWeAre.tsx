import { WhoWeAreStyled } from './styled'

/**
 * @component
 * @name WhoWeAre
 * @description Defines the container for the layout of who we are section.
 * @namespace components components/Layout/WhoWeAre
 * @param {element} children Component(s) that may appear inside the layout.
 *
 */

const WhoWeAre: React.FC = ({ children }) => {
  return <WhoWeAreStyled>{children}</WhoWeAreStyled>
}

export default WhoWeAre
