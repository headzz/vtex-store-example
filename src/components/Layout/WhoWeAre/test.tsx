import { render } from '@testing-library/react'

import WhoWeAre from './WhoWeAre'

describe('<WhoWeAre />', () => {
  it('should render component that wraps WhoWeAre on page', () => {
    const { container } = render(<WhoWeAre />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
