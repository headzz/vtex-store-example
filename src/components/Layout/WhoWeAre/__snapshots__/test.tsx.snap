// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[
  `<WhoWeAre /> should render component that wraps WhoWeAre on page 1`
] = `
.c0 {
  background: linear-gradient( 90deg,#ffffff 50%,#F2F2F2 50% );
  padding: 6.4rem 0;
  width: 100%;
}

<section
  class="c0"
/>
`
